webpack/assets/javascripts/react_app/components/ResourceErrors/index.js

```diff
@@ -0,0 +1,11 @@
+import { connect } from 'react-redux';
+
+import ResourceErrors from './ResourceErrors';
+
+import * as resourceErrorsActions from './ResourceErrorsActions';
+
+import reducer from './ResourceErrorsReducer';
+
+export const reducers = { resourceErrors: reducer };
+
+export default connect(null, resourceErrorsActions)(ResourceErrors);
```

Comment:
Redux hooks is about to be merged :) 
How about using it instead of `connect`? especially because there's no `mapStateToProps` here